test/functional/test-log.js

```diff
@@ -673,5 +674,33 @@ describe('Logging', () => {
       expect(duplicate.associatedElement).to.equal(error.associatedElement);
     });
   });
+
+  describe('embed error', () => {
+    let sandbox;
+    let iframe;
+    let element;
+
+    beforeEach(() => {
+      sandbox = sinon.sandbox.create();
+      iframe = document.createElement('iframe');
+      document.body.appendChild(iframe);
+    });
+
+    afterEach(() => {
+      sandbox.restore();
+      document.body.removeChild(iframe);
+    });
+
+    it('should return logger for user-error', () => {
+      expect(user().suffix_).to.equal(USER_ERROR_SENTINEL);
```

Comment:
you can use 
```
const error = user().createError();
expect(isUserErrorEmbed(error.message)).to.be.false;
```

also add test to make sure you not not creating extra identical logger

```
expect(user()).to.equal(user(element1));
expect(user()).to.equal(user(element2));
```